srcpkgs/lua-language-server/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'lua-language-server'
+pkgname=lua-language-server
+version=3.5.0
+revision=1
+create_wrksrc=yes
+hostmakedepends="ninja"
+short_desc="Lua LSP implementation written in Lua"
+maintainer="icp <pangolin@vivaldi.net>"
+license="MIT"
+homepage="https://github.com/sumneko/lua-language-server"
+changelog="https://github.com/sumneko/lua-language-server/blob/master/changelog.md"
+distfiles="${homepage}/releases/download/${version}/${pkgname}-${version}-submodules.zip"
+checksum=b8dbbd7834fb338a36426b11f8f2cac7c995e7ccf846ac0a5a02b8c7d3b55344
+
+do_build() {
+	ninja -C 3rd/luamake -f compile/ninja/linux.ninja
+	./3rd/luamake/luamake -platform ${XBPS_TARGET_MACHINE} rebuild
+}
+
+do_install() {
+	vinstall main.lua 644 usr/lib/${pkgname}
+	vinstall debugger.lua 644 usr/lib/${pkgname}
+	vinstall changelog.md 644 usr/lib/${pkgname}
```

Comment:
this should be `vdoc`ed

Reply:
But the `changelog.md` is required to be in the lib dir to run the `lua-language-server --version` command.
https://github.com/sumneko/lua-language-server/blob/3.5.0/script/version.lua#L3-L7

Reply:
ok, nevermind then